src/components/look-controls.js

```diff
@@ -123,9 +126,11 @@ module.exports.Component = registerComponent('look-controls', {
     window.addEventListener('mouseup', this.onMouseUp, false);
 
     // Touch events.
-    canvasEl.addEventListener('touchstart', this.onTouchStart);
-    window.addEventListener('touchmove', this.onTouchMove);
-    window.addEventListener('touchend', this.onTouchEnd);
+    if (this.data.touchEnabled) {
```

Comment:
We also have to handle the dynamic case when the flag changes at runtime. We probably need a method `updateTouchListeners` that contains this logic and its called from here and the `update` method.

Reply:
When the `enabled` flag is updated at runtime, a grab cursor is shown or hidden accordingly. I'm not sure what would need to be updated, other than the behavior that the flag controls (updated in commit 1d6930b61ac66d080bfe57eada841a032fd6e6e8). Would the behavior change accordingly when the flag is set at runtime?

Reply:
I think this is taken care of, given the last set of changes, yes?